api/server/router/system/system_routes.go

```diff
@@ -126,6 +126,9 @@ func (s *systemRouter) getInfo(ctx context.Context, w http.ResponseWriter, r *ht
 		if versions.GreaterThanOrEqualTo(version, "1.42") {
 			info.KernelMemory = false
 		}
+		if versions.LessThan(version, "1.49") {
```

Comment:
Turns out we did _not_ yet update API version; v1.49 already shipped with v28.1, so looks like we may have to update to v1.50 if we want to include this in v28.2;


- https://github.com/docker/docs/pull/22653
- https://github.com/moby/moby/pull/49718

Reply:
Ah, good catch!